extending/provider/implementation.rst

```diff
@@ -111,35 +63,39 @@ The following example implementation of ``refresh()`` updates an EBS volume in A
          setState(volume.stateAsString());
          setVolumeType(volume.volumeTypeAsString());
 
-         Ec2Client client = createClient(Ec2Client.class);
-
          DescribeVolumeAttributeResponse responseAutoEnableIo = client.describeVolumeAttribute(
-             r -> r.volumeId(getId())
-                 .attribute(VolumeAttributeName.AUTO_ENABLE_IO)
+             r -> r.volumeId(getId()).attribute(VolumeAttributeName.AUTO_ENABLE_IO)
          );
 
          setAutoEnableIo(responseAutoEnableIo.autoEnableIO().value());
+
+         return true;
      }
 
-**create(GyroUI ui, State state)**
+Implementing create
+===================
+
+The ``create(GyroUI ui, State state)`` method is called to create a resource. Gyro will call this method for every resource
+it determines needs to be created.
 
-The ``create(..)`` method is called by Gyro when it determines that it should create a resource. Implementations should
-create the resource and update any unique ID fields on the current object instance that will be necessary to query for
-the resource ``refresh()`` method.
+Implementations should create the resource and update any output properties after creation. At a minimum the id
```

Comment:
You may want to clarify "create the resource" here. The implementation should create a new Java instance from the providers resource library and populate it using the getters from the Gyro Java model?